docs/content/docs/ops/production_ready.md

```diff
@@ -81,4 +81,11 @@ It is a single point of failure within the cluster, and if it crashes, no new jo
 
 Configuring [High Availability]({{< ref "docs/deployment/ha/overview" >}}), in conjunction with Apache Zookeeper or Flinks Kubernetes based service, allows for a swift recovery and is highly recommended for production setups.
 
-{{< top >}}
\ No newline at end of file
+### Secure Flink Cluster Access
+
+To prevent potential security vulnerabilities, such as arbitrary code execution, 
```

Comment:
I think the language here can be more explicit. Arbitrary code execution is not a potential vulnerability.
Flink has been designed for arbitrary, remote code execution -- it is not an accident ;) 
Check also this FAQ entry for how we've worded this in the past: https://flink.apache.org/what-is-flink/security/#during-a-security-analysis-of-flink-i-noticed-that-flink-allows-for-remote-code-execution-is-this-an-issue